runtime/tests/runtime_integration_tests/test_ocp_lock/test_derive_mek.rs

```diff
@@ -0,0 +1,217 @@
+// Licensed under the Apache-2.0 license
+
+use caliptra_api::mailbox::{
+    CommandId, MailboxReq, MailboxReqHeader, OcpLockDeriveMekReq, OcpLockDeriveMekResp,
+};
+use caliptra_error::CaliptraError;
+use caliptra_hw_model::{HwModel, ModelError};
+
+use crate::test_ocp_lock::InitializeMekSecretParams;
+
+use super::{boot_ocp_lock_runtime, OcpLockBootParams};
+
+use zerocopy::{FromBytes, IntoBytes};
+
+/// Temporary checksum. This will be calculated from the full keyladder.
+const EXPECTED_MEK_CHECKSUM: [u8; 16] = [
+    243, 22, 202, 155, 137, 240, 133, 174, 210, 161, 117, 101, 142, 173, 20, 197,
+];
+
+// TODO(clundin): Follow up with the following test cases:
+//
+// These follow ups are pending an independently constructed key ladder / splitting into smaller
+// PRs.
+//
+// * Release MEK and validate MEK contents.
+// * Validate MEK checksum is correct.
+// * MEK and MEK checksum are the same after a warm reset.
```

Comment:
It would be good to have a hitless update test as well to make sure we don't distort the key when firmware with a different measurement is loaded